skeleton-esnext-webpack/package.json

```diff
@@ -107,6 +107,7 @@
     "@easy-webpack/config-uglify": "^2.1.0",
     "@easy-webpack/core": "^1.3.2",
     "aurelia-tools": "^0.2.4",
+    "babel-eslint": "^7.1.0",
```

Comment:
It is necessary in order to get linting working in js files in sublime text 3 (and other idé's?).